### DISCUSSION TITLE
Discord?

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#317] [Q&A] [answered]
Is there a discord for gituhb/iptv? If not, would it be worth it having one made? It might help with communication and development.

**Comment by BellezaEmporium** [comment, marked as answer]
There's no official iptv-org discord server.

Feel free to create a communitary server for this repository if you feel like it =)

**Comment by AntiPontifex** [comment]
I'll see what's involved :)

**Comment by AntiPontifex** [comment]
Update: I have JUST created a discord called IPTV on Github. How do I invite y'all to join: please let me know.

**Reply by BellezaEmporium** [reply]
Just share the invite link here. click on the name of your server, and "invite link"

**Comment by AntiPontifex** [comment]
https://discord.gg/DzTYhCJK

**Comment by AntiPontifex** [comment]
Thanks, as always, for the help LaneSh4d0w.

**Comment by AntiPontifex** [comment]
For any and all: the Discord channel is up and running; if interested, please follow the link above. Hopefully, this channel will prove to be beneficial to the community :)

**Comment by AntiPontifex** [comment]
New Link: https://discord.gg/QrDAj4WK

**Reply by savarela88** [reply]
already expired, can you share a new link to the discord? tks!

**Reply by BellezaEmporium** [reply]
https://discord.gg/3aBbzbB9

**Reply by AntiPontifex** [reply]
Thanks, Lane: you're a gentleman and a scholar.

**Comment by thomas-kichelm** [comment]


**Reply by BellezaEmporium** [reply]
https://discord.gg/knZTgCV6

**Comment by haaaaaaanson** [comment]
hi can i have a invitation?

**Comment by jeffjones217** [comment]
Is there a new invite link? Last one expired

**Comment by ggdagg** [comment]
Is it possible to have a new invite link ? All previous links are expired

**Reply by Dum4G** [reply]
https://discord.gg/wXvRZV4CWR
